Tests/Domain/ReleaseEvents/ReleaseEventTypesTests.cs

```diff
@@ -0,0 +1,22 @@
+using VocaDb.Model.Domain.ReleaseEvents;
+
+namespace VocaDb.Tests.Domain.ReleaseEvents;
+
+[TestClass]
+public class ArtistEventTypesTests
+{
+	[DataRow(0, EventCategory.Unspecified)]
+	[DataRow(1, EventCategory.AlbumRelease)]
+	[DataRow(2, EventCategory.Anniversary)]
+	[DataRow(4, EventCategory.Club)]
+	[DataRow(8, EventCategory.Concert)]
+	[DataRow(16, EventCategory.Contest)]
+	[DataRow(32, EventCategory.Convention)]
+	[DataRow(64, EventCategory.Other)]
+	[DataRow(128, EventCategory.Festival)]
+	[TestMethod]
+	public void Value(int expected, EventCategory actual)
+	{
+		((int) actual).Should().Be(expected);
+	}

```

Comment:
Since the values of the `EventCategory` enum are saved as a string in [DB](https://github.com/VocaDB/vocadb/blob/d7e30980f28a03c95f06877e4d5c7075fc72ac80/VocaDb.Migrations/Migrations.cs#L717-L736), and not as a bit field, you should write tests against names instead of numbers (and can you please make sure that?). See [PVServiceTests.cs](https://github.com/VocaDB/vocadb/blob/94e53c2bbf075415a42cb4859e46bbbee3bd4229/Tests/Domain/PVs/PVServiceTests.cs).

Reply:
Forgot to ask: if they are stored as strings, there should be no problem with switching the order order of values in the enum class, right? Currently, when selecting event type, "Festival" comes last, while I'd like it to come immediately after "Contest" (I think it makes more sense this way)

Reply:
Switching the order may work if and only if you are 100 percent sure that they are never stored as numbers. It would be disastrous if you accidentally changed it. I would suggest switching the order on the frontend instead, just like [ArtistTypesDropdownKnockout](https://github.com/VocaDB/vocadb/blob/9447fc42fed913e32b2c9f0d95eed103ce2a2545/VocaDbWeb/Scripts/Components/Shared/Partials/Artist/ArtistTypesDropdownKnockout.tsx), because it's more flexible. You could define a varaible like:
```tsx
const eventCategoryOrders: Record<EventCategory, number> = {
    [EventCategory.Unspecified]: 1,
    [EventCategory.AlbumRelease]: 2,
   // ...
}
```